modyn/trainer_server/internal/trainer/pytorch_trainer.py

```diff
@@ -306,7 +308,7 @@ def train(self) -> None:  # pylint: disable=too-many-locals, too-many-branches
                             loss = torch.dot(self._criterion_nored(output, target), weights / weights.sum())
                         else:
                             loss = self._criterion(output, target)
-
+                        loss = loss / self.gradient_accumulation_steps
```

Comment:
currently gradient accumulation steps can be None, so we should probably only do this if it is not none